hubspot3/test/test_tickets.py

```diff
@@ -49,3 +57,207 @@ def test_get_all_tickets():
     assert tickets
     assert isinstance(tickets, list)
     assert len(tickets) > 1
+
+
+@pytest.mark.parametrize(
```

Comment:
This is not the way I'm used to seeing nested structures in Python, not sure if PEP8 has a hard-saying in this. Usually it is written as:

```python
[(None, []),
 ("lead_source", ["lead_source"]),
 (["hs_analytics_last_url", "hs_analytics_revenue"],
  ["hs_analytics_last_url", "hs_analytics_revenue"]
  )
 ]
```

Notably, the optional trailing commas are very much a Lua thing and not so common in Python.

Reply:
I copied from contacts, so kept the structure, but I think this one is OK to change